README.md

```diff
@@ -0,0 +1,61 @@
+# urdf-packager
+
+A small util to turn a robot model into a single self-contained archive.
+
+## Use
+
+### Env setup
+
+Create a ROS 2 ready environment:
+
+```bash
+lxc launch ubuntu:22.04 urdf-exporter --config=user.user-data="$(curl -L https://gist.githubusercontent.com/artivis/1fcfcc38f1cb8c087e27ebda5484f2ef/raw/ros-bare.cloud-init.yaml)"
+```
+
+Shell inside the environment as the `ubuntu` user:
+
+```bash
+lxc exec urdf-exporter --mode interactive -- /bin/sh -xac $@ubuntu - exec /bin/login -p -f
+```
```

Comment:
Does it serve any purpose to shell into an ubuntu user?

Reply:
Stop doing everything as root !